vpcs.go

```diff
@@ -214,3 +231,35 @@ func (v *VPCsServiceOp) Delete(ctx context.Context, id string) (*Response, error
 
 	return resp, nil
 }
+
+func (v *VPCsServiceOp) ListMembers(ctx context.Context, id string, request *VPCListMembersRequest, opt *ListOptions) ([]*VPCMember, *Response, error) {
```

Comment:
Golang question: Is it a paradigm that interface functions return pointers and not actual values?

Reply:
Not necessarily, but it's the general pattern we're using in godo.